src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public interface Figure {
+    int figureArea();
```

Comment:
All our classes and interfaces should have a single purpose - draw() and getArea() methods are not conceptually close to each other.
Create another interface AreaCalculator for getArea() method